rke/templates/templates.go

```diff
@@ -189,12 +190,15 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.21.14-rancher1-1 <1.22.0-rancher0":    aciv523,
 			">=1.22.0-rancher0 <1.22.11-rancher1-1":    aciv500,
 			">=1.22.11-rancher1-1 <1.22.16-rancher1-1": aciv523,
-			">=1.22.16-rancher1-1 <1.23.0-rancher0":    aciv5234,
+			">=1.22.16-rancher1-1 <1.22.17-rancher1-2": aciv5234,
+			">=1.22.17-rancher1-2 <1.23.0-rancher0":    aciv5235,
```

Comment:
Update the ranges to account to addressing the previous comment.

Reply:
Resolved in [afe75d3](https://github.com/rancher/kontainer-driver-metadata/pull/1044/commits/afe75d357ed380cc14b9c708494309cc54fd52dc)